cdap-docs/build.sh

```diff
@@ -26,7 +26,7 @@
 # Builds each of these individually, and then packages them into a single zip file for distribution.
 # _common directory holds common files and scripts.
 
-source vars
+source ./vars
 source _common/common-build.sh
```

Comment:
"_common/" is a relative path already.
vars implies ./vars (./ is implicit). It needs to be spelled out when we shell out through a bamboo task.